### DISCUSSION TITLE
[@types/coingecko-api] No data type for CoinsFetchData.ticker

### DISCUSSION THREAD

**Discussion by JonathanTheZero** [DefinitelyTyped/DefinitelyTyped#57416] [Issues with a @types package]
Right now the definition for the `CoinsFetchData` in the @types/coingecko-api package looks like this:
```TS
interface CoinsFetchData {
    //...
    tickers: [];
}
```
What I believe is that the author either didn't know what type to expect or it heavily depends on the input. In the current version, however, this will be interpreted as the property always being an empty array, hence it's not possible to work with the data. I sadly don't have any in-depth knowledge of this specific API so an intermediate fix may be to type it as `any[]`. 

I will look into it and open a pull request in the near future with updated types if no one's faster than me.

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
Hi. I am not sure, maybe documentation is wrong but the parameter is a type of boolean based on documentation.

https://github.com/miscavage/CoinGecko-API#coinsfetch
